lib/wallaroo/core/topology/router.pony

```diff
@@ -896,6 +911,48 @@ class val LocalPartitionRouter[In: Any val,
       Fail()
     end
 
+  fun rebalance_steps_shrink(names_bs: Array[(String, OutgoingBoundary)],
```

Comment:
Maybe rename `name_bs` to `targets` or something like that?

Reply:
My original was `names_boundaries` but that felt a bit too long.  I was aiming for a name that describes the tuples in the array.  `names_bounds` could shave a few chars off that.  ... Hm, `target_workers` feels alright, what do you think?